include/swift/AST/DiagnosticEngine.h

```diff
@@ -750,13 +750,17 @@ namespace swift {
       // If the serialized diagnostics file not available,
       // fallback to the `YAML` file.
       if (llvm::sys::fs::exists(filePath)) {
-        localization = std::make_unique<diag::SerializedLocalizationProducer>(
-            filePath.str());
+        auto file = llvm::MemoryBuffer::getFile(filePath);
+        localization =
+            std::make_unique<diag::SerializedLocalizationProducer>(file.get());
       } else {
         llvm::sys::path::replace_extension(filePath, ".yaml");
-        assert(llvm::sys::fs::exists(filePath));
```

Comment:
I was using assertion, but this will fail with other languages that don't have translation files yet. So, I think we should fallback to default messages until we remove messages from `.def` files and then we can fallback to messages from the English file i.e. (`en.db`)